code.user.js

```diff
@@ -1313,11 +1321,11 @@
                         const callback = () => setTimeout(() => next(), getRandomInt(1000, 1500));
 
                         if (success) {
-                            logDOM(`${padLeft} - ${itemName} listed for ${formatPrice(market.getPriceIncludingFees(task.sellPrice))}, you will receive ${formatPrice(task.sellPrice)}.`);
+                            logDOM(`${padLeft} - ${itemName}${getAmountString(task.item.amount)} listed for ${formatPrice(market.getPriceIncludingFees(task.sellPrice) * task.item.amount)}, you will receive ${formatPrice(task.sellPrice * task.item.amount)}.`);
```

Comment:
I think this would be better to have in front. In your example: 34x Hexagonana
We can probably also avoid the function, it's pretty much a one-liner, e.g. const amount = task.item.amount == 1 ? '' : task.item.amount + 'x ';

Reply:
@Nuklon, done
![image](https://github.com/user-attachments/assets/88d2acfa-a27b-4349-9cc7-032f435a37c7)

Commit 494a094